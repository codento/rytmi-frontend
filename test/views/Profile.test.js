jest.mock('lodash', () => ({
  orderBy: jest.fn(() => {
    return [{
      name: 'Vue.js',
      knows: 0,
      wantsTo: 2,
      id: 7
    }]
  })
}))

import { shallow, createLocalVue } from 'vue-test-utils'
import View from '../../src/views/Profile.vue'
import lodash from 'lodash'

describe('Profile.test.js', () => {
  let cmp
  let $lodash = lodash
  beforeEach(() => {
    let localVue = createLocalVue()
    cmp = shallow(View, {
      localVue,
      mocks: {
        $lodash
      }
    })
  })
  it('Template is correct', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
