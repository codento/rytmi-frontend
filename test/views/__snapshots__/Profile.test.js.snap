// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile.test.js Template is correct 1`] = `
<div
  class="animated fadeIn"
>
  <b-row>
    <b-col>
      <!---->
    </b-col>
     
    <b-col>
      <b-card
        header="Taidot <i class=\\"float-right fa fa-edit\\"></i>"
      >
        <div
          class="row mb-1"
        >
          <!---->
        </div>
      </b-card>
       
      <b-card
        header="TyÃ¶kokemus"
      >
        <!---->
        <!---->
        <!---->
        <!---->
      </b-card>
    </b-col>
  </b-row>
</div>
`;
